public/app/features/auth-config/AuthConfigPage.tsx

```diff
@@ -75,6 +81,17 @@ export const AuthConfigPageUnconnected = ({ providerStatuses, isLoading, loadSet
     reportInteraction('authentication_ui_provider_clicked', { provider: provider.type });
   };
 
+  const getName = (providertype: string) => {
```

Comment:
@alexanderzobnin this is the way i got the displayname for the configuration currently. Let's expand on this after wards to make it better perhaps

Reply:
I would rather put name into `getConfigHook()` by extending `AuthProviderStatus`. The idea is to keep this component generic and independent from providers and let any information required here be returned by the provider's component.

Reply:
you do not think it would eventually be too much information that would be taken from that ?

Reply:
@alexanderzobnin updated to use the confighook for fetching the name if it exists